assets/RFC003-EthereumBitcoin-AtomicSwap.puml

```diff
@@ -0,0 +1,132 @@
+' DIAGRAM #########################################
+' RFC003 atomic swap
+' #################################################
+@startuml
+' #################################################
+' SETTINGS: color settings of diagram
+' #################################################
+skinparam sequence {
+	BorderColor black
+	ArrowColor black
+	ActorBorderColor black
+	LifeLineBorderColor black
+	LifeLineBackgroundColor white
+	
+	ParticipantBorderColor black
+	ParticipantBackgroundColor white
+	ParticipantFontColor black
+	
+	ActorBackgroundColor white
+	ActorFontColor black
+}
+' #################################################
+' ACTORS
+' #################################################
+actor Alice
+participant "Alice\nComit-Node" as AC #lightgreen
+participant "Alice\nETH-Client" as AETH #lightcyan
+participant "Alice\nBTC-Client" as ABTC #lightgoldenrodyellow
+participant "Ethereum\nNetwork" as ETH #lightcyan
```

Comment:
Maybe keep it `Alpha`/`Beta` to keep it generic and aligned with the RFC.

Reply:
I actually don't like Alpha and Beta ledger. I put the specific chains on purpose because it makes it way easier to understand it (because one can relate to something). However, probably we need a version of the diagram that is more abstract to be added to RFC003.

Reply:
I actually changed to it Alpha and Beta in the end. You are right, this diagram should just depict the content of the RFC. I might add additional, implementation specific diagrams later.